amplify/backend/function/fetchTokenFromChain/src/index.js

```diff
@@ -24,29 +29,44 @@ const baseToken = {
   colonies: null,
 };
 
-const setEnvVariables = async () => {
+const setEnvVariables = async (network) => {
   const ENV = process.env.ENV;
+
   if (ENV === 'qa') {
+    let chainRpcParam = 'chainRpcEndpoint';
+
+    switch (network) {
+      case BINANCE_NETWORK.shortName:
+        chainRpcParam = 'bnbRpcEndpoint';
+        break;
+      case ETHEREUM_NETWORK.shortName:
+      default:
+      // Use default chainRpcParam ie Ethereum to set `rpcURL`
```

Comment:
Shouldn't this switch also happen in `prod` and not just in QA?

Reply:
so at the moment we only have the QA environment up and running and a proper `prod` isnt yet running.

I've added it here in this case as this is the only condition that is checked so far ie such as what happens below. This is so that there is no confusion when we get around to adding in the case `if (ENV = 'prod')`

Reply:
True! Then perhaps we could add a note or a todo comment to remind us to update this once we have a prod env?

Reply:
I dont personally think this is necessary as I imagine we will just copy the entire `qa` if statement to the `prod` env because as you rightly pointed out this will need to be set there and I dont see anything changing